README.rst

```diff
@@ -97,7 +97,7 @@ Contribute
 - `Contributing guidelines <https://github.com/PlasmaControl/DESC/blob/master/CONTRIBUTING.rst>`_
 - `Issue Tracker <https://github.com/PlasmaControl/DESC/issues>`_
 - `Source Code <https://github.com/PlasmaControl/DESC/>`_
-- `Documentation <https://desc-docs.readthedocs.io/>`_
+- `Documentation <https://desc-docs.readthedocs.io/en/latest/>`_
```

Comment:
I'd prefer this to be stable as then it is consistent with our pip package

Reply:
ok

Reply:
```suggestion
- `Documentation <https://desc-docs.readthedocs.io/en/stable/>`_
```